frontend/src/metabase-lib/v1/parameters/utils/parameter-values.js

```diff
@@ -33,6 +37,8 @@ export function getValuePopulatedParameters({
   values = {},
   defaultRequired = false,
   collectionPreview = false,
+  // should be a better name for it
+  localDashboardParameters = {},
```

Comment:
I think `localDashboardParameters` are not needed and are side affect of changing `getParameterValue`. Could you pass `null` for `lastUsedParameterValue` and remove logic to reseting the parameters? If not, what is the issue that we need to solve here?

Reply:
there are several edge cases with the logic that's why I added this override for sure, but let me double check maybe it's not needed anymore

Reply:
@ranquild CI is green so you were right, nice catch!